Simplified local installation from sources (#5925)
kyma-project/kyma: docs/kyma/04-02-local-installation.md

     ```bash
     kyma install
     ```

>      >**NOTE** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling `kyma install --source {KYMA_RELEASE}`.

Reviewer: ```suggestion
     >**NOTE:** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling `kyma install --source {KYMA_RELEASE}`.
```

  4. Install Kyma from sources. Run:

     ```bash
     kyma install --source local
     ```

>      >**NOTE** By default the Kyma sources located in your [GOPATH](https://github.com/golang/go/wiki/GOPATH) will be used. If you want to use a specific source folder use the folder as a parameter when calling `kyma install --source local --src-path {YOUR_KYMA_SOURCE_PATH}`.

Reviewer: ```suggestion
     >**NOTE:** By default, the installation uses sources located under your [GOPATH](https://github.com/golang/go/wiki/GOPATH). If you want to use a specific source folder, use it as a parameter when calling `kyma install --source local --src-path {YOUR_KYMA_SOURCE_PATH}`. 
```
